datasets/german_common_crawl/README.md

```diff
@@ -0,0 +1,211 @@
+---
+YAML tags:
```

Comment:
We expect to have the YAML tags here, not JSON (even though the tagger app allows to save as JSON)
Can you replace the tags with the YAML instead ?
You can fin the YAML at the top right of the tagger app by clicking `expand yaml tags`